site/gatsby-site/cypress/e2e/unit/functions/apiRequest.cy.js

```diff
@@ -1,62 +0,0 @@
-const apiRequest = require('../../../../../realm/functions/apiRequest');
```

Comment:
do we have a corresponding test for this in playwright? or is this just on it's way out in general?

Reply:
Deleted soon with the magic links stuff, and we are very very unlikely to modify any of those functions.